src/components/ChangelogSnippet/ChangelogSnippet.astro

```diff
@@ -0,0 +1,60 @@
+---
+import { SvgArrowRight2, Typography } from "@chainlink/blocks"
+import { searchClient } from "@algolia/client-search"
+import ChangelogCard from "./ChangelogCard.astro"
+import { ALGOLIA_INDEX, type ChangelogItem } from "./types"
+import styles from "./ChangelogSnippet.module.css"
+import { getSecret } from "astro:env/server"
+
+interface Props {
+  query: string
+}
+
+const { query } = Astro.props
+
+const appId = getSecret("ALGOLIA_APP_ID")
+const apiKey = getSecret("PUBLIC_ALGOLIA_SEARCH_PUBLIC_API_KEY")
+
+if (!appId || !apiKey) {
+  throw new Error("Missing Algolia credentials")
+}
+
+const client = searchClient(appId, apiKey)
+
+const req = await client.search({
+  requests: [
+    {
+      indexName: ALGOLIA_INDEX.CHANGELOG,
+      query,
+    },
+  ],
+})
+
+const firstResult = req.results[0]
+const results = "hits" in firstResult ? (firstResult.hits as ChangelogItem[]) : []
+
+// logs are returned sorted by created_at DESC
+const latestLog = results[0]
+---
+
+{
+  results.length > 0 && (
+    <section class={styles.container}>
```

Comment:
Not enough padding after this section

<img width="1264" height="469" alt="image" src="https://github.com/user-attachments/assets/e5577204-0076-49af-8dd6-c38a57b3d443" />